src/sage/interfaces/maxima.py

```diff
@@ -31,6 +31,7 @@
 
 This is the interface used by the maxima object::
 
+    sage: from sage.interfaces.maxima import maxima
```

Comment:
do inform the readers of this documentation file about the existence of the library version.

Reply:
I second this

Reply:
done